problems/arraysWithLoops.js

```diff
@@ -66,15 +130,21 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+const findSecondSmallest = (numbs) =>{
+  let number = 0;
+  numbs.sort(function(a, b));
+
+}
 
 /**
  * Takes in array and returns the second largest number in the array
  * @param {number[]} nums
  * @returns {number} second largest value
  */
 
-function findSecondLargest() {}
+function findSecondLargest() {
+  points.sort(function(a, b){return b-a});
```

Comment:
points is not defined. 
Although sorting the input could solve this, try doing it with two variables instead called largest, and secondLargest